Makefile

```diff
@@ -41,10 +41,17 @@ MSBUILD=/cygdrive/c/Program\ Files/MSBuild/12.0/Bin/MSBuild.exe
 endif
 
 ifeq ($(uname), Darwin)
-executable=./src/ui/osx/TogglDesktop/build/Release/TogglDesktop.app/Contents/MacOS/TogglDesktop
+xcodebuild_command=xcodebuild $(XCODELEGACYSDK) \
+				  -scheme TogglDesktop \
+				  -project src/ui/osx/TogglDesktop/TogglDesktop.xcodeproj  \
+				  -configuration Release
+executable=$(shell $(xcodebuild_command) \
+			 -showBuildSettings \
+ 			| grep -w 'BUILT_PRODUCTS_DIR' \
+ 			| cut -d'=' -f 2)/TogglDesktop.app/Contents/MacOS/TogglDesktop
```

Comment:
Get `buildSettings` from Xcode, and export `BUILT_PRODUCTS_DIR`.

Ex:
```
executable=/Users/nghiatran/Library/Developer/Xcode/DerivedData/TogglDesktop-crbgfktryatxznfuieitjreuwxyo/Build/Products/Release/TogglDesktop.app/Contents/MacOS/TogglDesktop
```

Reply:
I'll test it out once more with clean build to be sure and then we'll merge if everything works ok.